c2corg_ui/static/js/viewdetails.js

```diff
@@ -168,6 +222,77 @@ app.ViewDetailsController.prototype.toggleTab = function(tab) {
 
 
 /**
+ * blablabla
+ * @export
```

Comment:
Do you really need to export this function?
It looks like a private function (the name should have a trailing "_")?